Travis autoupload guide.md

```diff
@@ -0,0 +1,43 @@
+# Travis autoupload guide
+
+With this setup, you can automatically deploy your extension when creating a tag.
+
+## Setup
+
+0. Enable your repo on travis
+0. Create a `.travis.yml` file and set your `EXTENSION_ID`:
+
+    ```yaml
```

Comment:
Good thing you noticed, though, because the other one is not indented and it looks weird now. Fixing...